how-to/customize-workspace/client/src/modules/integrations/pages/integration-provider.ts

```diff
@@ -0,0 +1,328 @@
+import {
```

Comment:
One thing I am worried about is module size. 
As we are actually importing the module it will be pulling in another copy of the sdk?

Reply:
Update the code to use types, but this needed to add `getPlatform` method to integration helper, shaved 2mb of each bundle :)